### ISSUE TITLE
basic info to run the mcmot bytetrack 

### ISSUE THREAD

**Issue by hlmhlr** [CaptainEven/MCMOT-ByteTrack#8]
Hi, 
Can you please guide for the following queries:

1. I have the output of detector in .txt file, in which part of the code I should mention the class ID indexing so that it should pick the specific number of classes?
2. Do I need to create the .json file for multi-class? If yes, then in which part of the code do I have to mention the class ids? Let's say I am following the MOT-17 format and trying to use only test data.
3. Can I run the tracker without yolox pre-trained file? Means, simply giving the detection as input to the ByteTrack for tracking for multiple classes.

I shall be thankful for your response. 

Kind Regards,

**Comment by CaptainEven** [comment]
Sorry for the loss of  detailed instructions! Because i'm busy at work recently. I will give some detailed instructions when i'm free.

**Comment by arnaud-nt2i** [comment]
@CaptainEven ,up,
do you have time to answer the questions ?
